charts/cluster-proportional-autoscaler/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 name: cluster-proportional-autoscaler
 version: 1.0.0
```

Comment:
Actually - I assume this should be bumped as well?

Reply:
I would make it 1.0.2